hackathon_site/dashboard/frontend/src/components/inventory/Item/Item.tsx

```diff
@@ -18,8 +18,8 @@ const Item = ({ image, title, total, currentStock }: ItemProps): ReactElement =>
     const stock = !currentStock
         ? "OUT OF STOCK"
         : userType === "participant"
-        ? currentStock + " IN STOCK"
-        : currentStock + " OF " + total + " IN STOCK";
+        ? Math.max(currentStock, 0) + " IN STOCK"
```

Comment:
this will show the message `0 IN STOCK` but we actually want it to say `OUT OF STOCK`, could you make the changes to allow that?